src/app/video-courses/components/video-courses-list/video-courses-list.component.ts

```diff
@@ -1,20 +1,18 @@
 import { Component, OnInit } from '@angular/core';
-import { VideoCoursesService } from '../services/video-courses.service';
-import { Course } from '../shared/models/course.model';
-import { FilterPipe } from './filter.pipe';
+import { VideoCoursesService } from '../../services/video-courses.service';
+import { Course } from '../../shared/models/course.model';
```

Comment:
in tsconfig.json in paths section "nice" path could be configured from src folder:
 "paths": {
      "@shared-cources/*": [
        "app/video-courses/shared*"
      ],
}
so path will look like `import { Course } from '@shared-cources/models/course.model';` in whole app, not depending on folder structure.